### PR TITLE
[Landcover Toolkit] Add tests for getSpectralBands()

### PR DESCRIPTION
**Pull request by gino-m**
Came across this while working on #74. Is this the right return type and docs here?

### CODE REVIEW

**Comment by gino-m** [comment]
@gorelick-google also, in the following:
https://github.com/google/earthengine-community/blob/a413c7a8f3bf2db9481736dd9dfde9e9aa18c234/toolkits/landcover/impl/bands.js#L65-L88

What EE type would `indices` be? If it's `ee.Array` this may need a tweak, since looking at docs `ee.Array` doesn't seem to have a `map()` operation (cmiiw).

File: toolkits/landcover/impl/bands.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ var SPECTRAL_INDEX_EXPRESSIONS = {
  *
  * @param {ee.Image} image The source image.
  * @param {!Array<string>} indices The list of indices to calculate.
- * @return {ee.Image} The updated image.
+ * @return {!Array<!ee.Image>} The new spectral index bands.
```
Comment (gorelick-google) [inline toolkits/landcover/impl/bands.js]: This is technically correct now, but not what's supposed to happen.

This should really return an Image, which means the map at the bottom (which is mapping over a regular javascript array) should be wrapped with an ee.Image constructor.  (Also the docs say it "adds" to the existing image; that's not true either, it should say: "Compute the specified..."

**Comment by gino-m** [comment]
> @gorelick-google also, in the following:
> https://github.com/google/earthengine-community/blob/a413c7a8f3bf2db9481736dd9dfde9e9aa18c234/toolkits/landcover/impl/bands.js#L65-L88
> 
> What EE type would `indices` be? If it's `ee.Array` this may need a tweak, since looking at docs `ee.Array` doesn't seem to have a `map()` operation (cmiiw).

@gorelick-google what about this one?

**Comment by gino-m** [comment]
We spoke - using `ee.List` here doesn't currently work due some combination of `ee.Dictionary`, image expressions, iterate and map. I'll roll back those change. Keeping them here for future use:
https://gist.github.com/gino-m/9d75e16a23c03b0cf4e47f590d54018c

**Pull request merged**